src/components/wallets/WalletsListItem.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <v-list-item :title="localWallet.cryptoName">
+    <v-list-item-subtitle>
+      <p>
+        {{ localWallet.type }} <b>{{ localWallet.symbol }}</b>
+      </p>
+    </v-list-item-subtitle>
+    <template v-slot:prepend>
+      <v-avatar>
+        <crypto-icon :class="classes.cryptoIcon" :crypto="localWallet.symbol" size="medium" />
+      </v-avatar>
+    </template>
+
+    <template v-slot:append>
+      <WalletBalance
+        :wallet-balance="{
+          balance: localWallet.balance,
+          currentFiatCurrency,
+          rate: localWallet.rate
+        }"
+      ></WalletBalance>
+      <v-checkbox hide-details v-model="localWallet.isVisible"></v-checkbox>
```

Comment:
It's a bad practice to mutate the Vue prop. This leads to bugs.

Better to use this approach:


```vue
<v-checkbox
  :model-value="store.getters['wallets/getVisibility'](symbol)"
  @update:model-value="store.commit('wallets/updateVisibility', $event.target.value)"
/>
```

Reply:
Not fixed